.github/workflows/cacerts-release.yaml

```diff
@@ -62,6 +63,8 @@ jobs:
           path: work
           branch: "cacerts-${{ steps.nss-latest-release.outputs.NSS_VERSION }}-${{ steps.figure-out-branch.outputs.BRANCH }}"
           base: ${{ steps.figure-out-branch.outputs.BRANCH }}
+          author: "Flatcar Buildbot <buildbot@flatcar-linux.org>"
+          committer: "Flatcar Buildbot <buildbot@flatcar-linux.org>"
```

Comment:
I'd argue that if git is not clean, we should fail the workflow instead of papering it over by creating a generic commit. If this happens, then it means that the action is not invoking `cleanup_repo` at the end of the script.

Reply:
Ok, I have dropped the last commit of author/committer.

BTW have just realized we have another git path issue in `cleanup_repo`, so fixed that as well.